sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py

```diff
@@ -143,8 +155,17 @@ def generate_events(pipeline, args, pipeline_options):
     topic.publish(AUCTION_EVENT)
     topic.publish(BID_EVENT)
 
+  # Large and comprehensive
```

Comment:
I think you can remove this comment block. As we discussed offline, ReadFromText does not work in streaming mode and WriteStringsToPubSub does not work in batch mode. This pipeline would nor because of that.

It is fine to do the publishing in a loop in the test code.

Reply:
Yes, I have used file system's open for local or GCS files.